pkg/component/runtime/manager_test.go

```diff
@@ -2493,7 +2498,36 @@ func TestManager_FakeShipper(t *testing.T) {
 							subErrCh <- fmt.Errorf("unit reported unexpected state: %v", unit.State)
 						}
 					} else {
-						subErrCh <- errors.New("unit missing: fake-input")
+						subErrCh <- errors.New("input unit missing: fake-input")
```

Comment:
Nit: should this be `fake-default`?
```suggestion
						subErrCh <- errors.New("input unit missing: fake-default")
```